flaskapp/blueprints/courses/routes.py

```diff
@@ -57,8 +56,8 @@ def all_courses():
     return render_template(
         "questions/courses.html",
         courses=_courses,
-        css_file="css/base.css",
-        css_file2="css/questions/courses.css",
+        css_files=["css/base.css", "css/questions/courses.css"],
+        js_files=["js/questions/remove_course.js"],
```

Comment:
Move js, css, html regarding courses into courses folder just like in blueprints